packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import * as notificationActions from '@suite-actions/notificationActions';
+import { fetchTransactionsThunk } from '@suite-common/wallet-core';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+import { FormattedCryptoAmount, Translation } from '@suite-components';
+import { SETTINGS } from '@suite-config';
+import { useActions } from '@suite-hooks';
+import { useTranslation } from '@suite-hooks/useTranslation';
+import { ExtendedMessageDescriptor } from '@suite-types';
+import { Pagination } from '@wallet-components';
+import { Checkbox, Icon, Switch, variables } from '@trezor/components';
+import { UtxoSelectionList } from '@wallet-components/UtxoSelectionList';
+import { useSendFormContext } from '@wallet-hooks';
+import { useBitcoinAmountUnit } from '@wallet-hooks/useBitcoinAmountUnit';
+import { TypedFieldError } from '@wallet-types/form';
+
+const Row = styled.div`
+    align-items: center;
+    display: flex;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const SecondRow = styled(Row)`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    margin-top: 12px;
+`;
+
+const DustRow = styled.div`
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-top: 6px;
+`;
+
+const GreyText = styled.div`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const DustDescriptionRow = styled(GreyText)`
+    margin: 6px 0 12px 0;
+`;
+
+const StyledSwitch = styled(Switch)`
+    margin: 0 14px 0 auto;
+`;
+
+const AmountWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-left: auto;
+    text-align: right;
+`;
+
+const MissingToInput = styled.div<{ isVisible: boolean }>`
+    visibility: ${({ isVisible }) => !isVisible && 'hidden'};
+`;
+
+const StyledPagination = styled(Pagination)`
+    margin-top: 20px;
+`;
+
+const Line = styled.div`
+    width: 100%;
+    height: 1px;
+    border-top: 1px solid ${({ theme }) => theme.STROKE_GREY};
+    margin: 12px 0 16px 0;
+`;
+
+interface Props {
+    close: () => void;
+}
+
+export const CoinControl = ({ close }: Props) => {
+    const [currentPage, setSelectedPage] = useState(1);
+
+    const { addToast, fetchTransactions } = useActions({
+        addToast: notificationActions.addToast,
+        fetchTransactions: fetchTransactionsThunk,
```

Comment:
please try not to rename, it's harder to refactor afterwards, just name it `fetchTransactionsThunk`

Reply:
I don't see a way how not to rename it when used with useActions 🤔

Reply:
Oh yea, that's shit, another reason why to ditch useActions